scripts/update-dependencies/project.json

```diff
@@ -0,0 +1,25 @@
+{
+  "version": "1.0.0-*",
+  "description": "Updates the repos dependencies",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "NETStandard.Library": "1.5.0-rc2-23911",
+    "Microsoft.CSharp": "4.0.1-rc2-23911",
+    "System.Runtime.Serialization.Primitives": "4.1.1-rc2-23911",
+    "Microsoft.DotNet.Cli.Build.Framework": "1.0.0-*",
+    "NuGet.Versioning": "3.4.0-rtm-0764",
+    "Newtonsoft.Json": "7.0.1",
+    "Octokit": "0.18.0",
+    "Microsoft.Net.Http": "2.2.29"
+  },
+  "frameworks": {
+    "netstandardapp1.5": {
+      "imports": [
+        "dnxcore50",
+        "portable-net45"
```

Comment:
Yes, the package that falls into this bucket is:  https://www.nuget.org/packages/Octokit/0.18.0, which we are using to create the PR on GitHub.  It has "portable-net45+wp80+win+wpa81" from looking at the package.

It happens to be working both on my machine and in the VSO build. I guess I haven't tried it on a Unix machine... Do you think I should add the full portable string?